src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+leads to it becoming a universal. However, how _exactly_ are we supposed to
+understand this? Is the mechanical process at work between the scented molecules
+and the non-scented molecules, or is it just between the scented molecules? To
+pose this question is, I think, to overstep the conceptual restrictions of
+Mechanism and mechanistic thought. The distinction between a scented molecule
+and a non-scented molecule does not obtain within a mechanistic framework
+because, as `mechanical objects`, they do not have different determinations
+(_i.e._ one is scented and the other is non-scented). Quite simply, scented
+molecules and non-scented molecules have an identical determinateness.
+
+### Interlude: How does the Science of Logic relate to External Reality?
+
+A further philosophical conundrum poses itself. Above, I said that the
+distinction between a scented molecule and a non-scented molecule does not
+obtain within a mechanistic framework. This mode of thinking about how the
+logical determinations are reflected in everyday occurrences suggests that the
+same object can be thought of within a mechanistic framework, as well as other
+frameworks. Take the human body, one could conceive of it purely in mechanistic
+terms and theories of its processes in terms of cause and effect, or one could
+conceive of it in organicist terms and conceive of its processes differently.
+Now, what is crucial about this example is that it works because mechanistic
+thinking is more reductive than organicist thinking – the reverse would not be
+possible. One could not conceive of a rock within an organicist framework. Thus,
+the possibility of an object to be conceived within a plurality of frameworks
+requires that the object is open to reductionism.
+
+Now, a further thought. This does not mean that it is equally correct to
+conceive of the same object through the prism of multiple frameworks. The human
+body is an organic thing and so to conceive of it as a `mechanical object` is to
+misconstrue it. It _can_ be done but you are failing to fully grasp what it is a
+for a human body to be what it is. Let us hold onto this reflection and turn
+back to the example of the diffusion of scent. Since we are within the realm of
+Mechanism, the simplest way that we have to conceive of the scented and the
+non-scented molecules is as `mechanical objects`[^2]: Of course, one could
+conceive of them as a `syllogism` or a `judgement`, but then they would cease to
+be conceived of as objects. Instead, they would be conceived of as concepts. It
+is plausible that one would have a more accurate conception of them through a
+chemical framework, as found in Chemism, which actually takes into account their
+different determinateness. But for now, they are `mechanical objects` whose
+determinateness is identical.
+
+### The Ideality of Communication
+
+Returning to the text and progressing with it, Hegel writes that: “But even in
```

Comment:
This kind of "returning to the text etc." signposting can become problematic. What if somebody adds a bunch of paragraphs or sections before this. This would break the original chain of thought and one would have to go into this paragraph in order to fix it. I think we should try to write in a way that is as self-contained and modular as possible, sticking to the topic at hand.

Reply:
Yep, makes sense. I have changed it to the more generic: "Looking at the second half of the first paragraph of `(a)The Formal Mechanical Process`"